services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/service/SegmentDatabaseService.java

```diff
@@ -0,0 +1,19 @@
+package com.futurewei.alcor.vpcmanager.service;
+
+import com.futurewei.alcor.common.db.CacheException;
+import com.futurewei.alcor.common.db.ICache;
+import com.futurewei.alcor.common.exception.DatabasePersistenceException;
+import com.futurewei.alcor.web.entity.SegmentWebResponseObject;
+import com.futurewei.alcor.web.entity.VpcWebResponseObject;
+
+import java.util.Map;
+
+public interface SegmentDatabaseService {
```

Comment:
There is another service interface called "SegmentService" but it also deals with database access, right? In this case, we should use names that match what it is exactly? 

A renaming suggestion:
(1) SegmentDatabaseService  => SegmentManagementService
(2) SegmentService  => SegmentKeyManagementService

Reply:
Let's talk about it